postgres-appliance/bootstrap/clone_with_wale.py

```diff
@@ -57,8 +60,7 @@ def fix_output(output):
     started = None
     for line in output.decode('utf-8').splitlines():
         if not started:
-            started = re.match(r'^name\s+last_modified\s+', line)
-            started = re.match(r'^name\s+last_modified\s+', line) or re.match(r'^name\s+modified\s+', line)
+            started = re.match(r'^(backup_)?name\s+(last_)?modified\s+', line)
```

Comment:
Make sure this matches up with the fix upstream so that we don't reintroduce the bug: https://github.com/zalando/spilo/pull/1026